cmd/gpu_plugin/README.md

```diff
@@ -242,3 +247,69 @@ We can test the plugin is working by deploying an OpenCL image and running `clin
       ----     ------            ----       ----               -------
       Warning  FailedScheduling  <unknown>  default-scheduler  0/1 nodes are available: 1 Insufficient gpu.intel.com/i915.
     ```
+
+
+## Issues and workarounds
+
+### Media workloads on multi-GPU setups
+
+#### Media issue
```

Comment:
nit: these `####` level subsections do not help readability anymore, IMO

Reply:
IMHO the issue and workaround/examples need to be separate, they are too long to be in single section.

Would this be better:
```
## Media workload issues on multi-GPU setups
### Workaround
```

Reply:
Looks better yes